README.md

```diff
@@ -59,6 +59,19 @@ The topics will be inferred by the browser. The browser will leverage a classifi
         * is secure
         * has a non-opaque origin
         * is the primary main frame or is its child iframe (i.e. not a fenced frame , not a pre-rendering page)
+
+* Topics can also be retrieved via request headers, and marked as observed and eligible for topics calculation via response headers.
+    * This is likely to be considerably more performant than using the JavaScript API.
+    * The request header will be sent on document requests when the list of topics is non-empty and the request is allowable (e.g., permission policy allows it, the context is secure, etc.).
+    * The request header can be sent along with fetch requests via specifying an option: `fetch(<url>, {browsingTopics: true})`.
+    * Redirects will be followed, and the topics sent in the redirect request will be specific to the redirect url.
+    * The request header will not modify state for the caller unless there is a corresponding response header. That is, the topic of the page won't be considered observed, nor will it affect the user's topic calculation for the next epoch. 
+    * The response header will only be honored if the corresponding request included the topics header (or would have included the header if it wasn't empty).
+    * The registrable domain used for topic observation is that of the url of the request.
+    * Example request header: `Sec-Browsing-Topics: 123;model=1;taxonomy=1;version=2, 2;model=1;taxonomy=1;version=2`
+        * This example has two topics, 123 and 2, along with their version information.
+    * Example response header: `Observe-Browsing-Topics: 1`
```

Comment:
(From @domenic) Should this be "?1", so that it's a structured field https://www.rfc-editor.org/rfc/rfc8941.html#section-3.3.6-4

Reply:
+1

Reply:
Whoops, thanks! Done.